TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,7 +104,7 @@ public function initializeArguments()
     /**
      * Renders an HTML img tag with a thumbnail image, created from a given image.
      *
-     * @param ImageInterface $image The image to be rendered as an image
+     * @param ImageInterface|null $image The image to be rendered as an image
```

Comment:
Just tried it again, can't reproduce. I could take out the `|null` fallback, but would still be interesting to know why it works on my system and not on yours. Maybe we can compare later or tommorrow.